packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -49,6 +49,29 @@ export class OpGroupingManager {
 		this.logger = createChildLogger({ logger, namespace: "OpGroupingManager" });
 	}
 
+	/**
+	 * Creates a new batch with a single message of type "groupedBatch" and empty contents.
+	 * @param referenceSequenceNumber - reference sequence number
+	 * @returns - IBatch with empty content
+	 */
+	public createEmptyGroupedBatch(referenceSequenceNumber: number): IBatch<[BatchMessage]> {
+		const serializedContent = JSON.stringify({
+			type: OpGroupingManager.groupedBatchOp,
+			contents: [],
+		});
+
+		return {
+			contentSizeInBytes: 0,
+			messages: [
+				{
+					referenceSequenceNumber,
```

Comment:
You need to pass `resubmittingBatchId` through to here and put it on the metadata, right?